source/mesh_deformation/fastscape.cc

```diff
@@ -0,0 +1,1054 @@
+    {
+
+      prm.declare_entry ("End time",
+                         /* boost::lexical_cast<std::string>(std::numeric_limits<double>::max() /
+                                              year_in_seconds) = */ "5.69e+300",
+                         Patterns::Double (),
+                         "The end time of the simulation. The default value is a number "
+                         "so that when converted from years to seconds it is approximately "
+                         "equal to the largest number representable in floating point "
+                         "arithmetic. For all practical purposes, this equals infinity. "
+                         "Units: Years if the "
+                         "'Use years in output instead of seconds' parameter is set; "
+                         "seconds otherwise.");
+
+      prm.enter_subsection ("Mesh deformation");
+      {
+        prm.enter_subsection ("Fastscape");
+        {
+          prm.declare_entry("Number of steps", "10",
+                            Patterns::Integer(),
+                            "Number of steps per ASPECT timestep");
+          prm.declare_entry("Maximum timestep", "10e3",
+                            Patterns::Double(0),
+                            "Maximum timestep for fastscape.");
+          prm.declare_entry("Vertical exaggeration", "3",
+                            Patterns::Double(),
+                            "Vertical exaggeration for fastscape's VTK file.");
+          prm.declare_entry("Additional fastscape refinement", "0",
+                            Patterns::Integer(),
+                            "Refinement level expected at surface to determine"
+                            "proper nx and ny values");
+          prm.declare_entry ("Use center slice for 2d", "false",
+                             Patterns::Bool (),
+                             "If this is set to true, then a 2D model will only consider the "
+                             "center slice fastscape gives. If set to false, then aspect will"
+                             "average the inner third of what fastscape calculates.");
+          prm.declare_entry("Fastscape seed", "1000",
+                            Patterns::Integer(),
+                            "Seed used for adding an initial 0-1 m noise to fastscape topography.");
+          prm.declare_entry("Surface resolution", "1",
+                            Patterns::Integer(),
+                            "This should be set to the highest ASPECT resolution level you expect at the surface.");
+          prm.declare_entry("Resolution difference", "0",
+                            Patterns::Integer(),
+                            "The difference between the lowest and highest resolution level at surface. So if three resolution "
+                            "levels are expected, this would be set to 2.");
+          prm.declare_entry ("Use marine parameters", "false",
```

Comment:
Is this to use marine parameters or to use the whole marine functionality? If the latter, update the name to reflect this.